src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -113,8 +114,9 @@ class CourtController(
     @RequestBody
     @Valid
     videoLinkBookingUpdateSpecification: VideoLinkBookingUpdateSpecification?
-  ) {
+  ): ResponseEntity<Void> {
     courtService.updateVideoLinkBooking(videoBookingId!!, videoLinkBookingUpdateSpecification!!)
+    return ResponseEntity.noContent().build()
```

Comment:
Bit rubbish but currently required due to [this](https://github.com/springfox/springfox/issues/3557) outstanding bug.

Might be worth looking at moving over to springdocs